### PR TITLE
Update docs for social login feature

### PR DESCRIPTION
**Pull request by nataliejeremy**
## Effect
PR includes the following changes:
- Updates /user-dashboard to include the google account info; general updates
- Updates /sso-organizations doc with a note that social login is disabled for orgs using SSO; adds link to SSO doc
- Updates /sso doc to add link to the sso-organizations doc

@ari-gold please review

### CODE REVIEW

File: source/_docs/sso-organizations.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Detailed information to enable SAML single sign-on for your organiz
 categories: [managing]
 tags: [platform, organizations]
 ---
-Single sign-on (SSO) allows users to authenticate against your Identity Provider (IdP) when logging into the Pantheon Dashboard.
+[Single sign-on](/docs/sso/) (SSO) allows users to authenticate against your Identity Provider (IdP) when logging into the Pantheon Dashboard.
```
Comment (ari-gold) [inline source/_docs/sso-organizations.md]: Not sure this link fits here

File: source/_docs/sso-organizations.md [hunk 2]
```diff
@@ -59,3 +59,6 @@ Open a support ticket with Pantheon with the following:
 ## Troubleshooting
 ### Cannot Authenticate with Username/Password When Creating a New Machine Token
 If you are a member of a SAML-enabled organization, and the password field does not disappear after you enter your username and password, you'll need to log out of your active session, log back in, and try again. This can occur if you have two accounts with different email addresses and have not logged out of an active session.
+
+### Cannot Login Using Google Account
+Members of organizations that use single sign-on are unable to login to Pantheon with their Google account, and must log in using their company email address.
```
Comment (ari-gold) [inline source/_docs/sso-organizations.md]: Social Login is disabled for users in organizations using single sign-on. SAML allows organizations to centrally manage authentication policies, so allowing Social Login could circumvent that policy.

File: source/_docs/user-dashboard.md [hunk 3]
```diff
@@ -8,28 +8,28 @@ tags: [platform]
...
-The Account tab is where you can update your user profile details, change your password, add and manage SSH keys, view billing details, add machine tokens, and delete your account.
+### Account Tab
+This is where you can update and manage your personal account details. Use the options on the left navigation menu to:
 
-##Team Management
-It's easy to add, manage, and delete your site's users. For complete details, see [Team Management](/docs/team-management/).
-![Team Management Window](/source/docs/assets/images/team-modal.png)
-##Pantheon for Agencies
+- Update your user profile details
+- Connect and disconnect your Google account
```
Comment (ari-gold) [inline source/_docs/user-dashboard.md]: add: (used for Social Login)

**Comment by ari-gold** [comment]
@nataliejeremy Thanks! LGTM.

File: source/_docs/user-dashboard.md (hunk 3 above)
Comment (nataliejeremy) [inline source/_docs/user-dashboard.md]: @ari-gold I don't see it called social login anywhere that's user facing...I only see Google account. Is that changing?

**Pull request merged**